sla_per_user_system_test.py

```diff
@@ -371,6 +372,13 @@ def test_read_throughput_1to5_ratio(self):
 
     def _two_users_load_througput_workload(self, shares, load, expected_shares_ratio=None):
         session = self.prepare_schema()
+
+        if is_tablets_feature_enabled(session=session):
+            session.execute("""
```

Comment:
There is `pre_create_keyspace` test parameter. 
Longevity test has a function `longevity_test.LongevityTest.run_pre_create_keyspace` (that inherits in SLA test)
This command can be added to the test yaml.
Or under `scylla-cluster-tests/configurations` and the used in the jenkins pipeline.
See example here:
https://github.com/scylladb/scylla-cluster-tests/blob/99d539bbdc3a4ec912511e761275f0c202fd9a30/jenkins-pipelines/manager/debian11-manager-install.jenkinsfile#L10.

This way prevents hardcoded cql command inside the code

Reply:
but with ```pre_create_keyspace``` and ```scylla-cluster-tests/configurations``` I cant use if is_tablets_feature_enabled

and adaptively create keyspace for Scylla versions with and without tablets

Reply:
yes, you still can handle this command when tablets are enabled.
for you decision. I think it is better to put it in test yaml (if you do not like configurations )